contents/blog/the-posthog-array-1-33-0.md

```diff
@@ -0,0 +1,124 @@
+
+
+<img src="https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/1_33_0-dashboard-permissions.png" alt="Screenshot: Dashboard permissions" />
+
+> 🎁 Dashboard permissions is a premium feature and requires a **PostHog Enterprise** license. [Learn more](/pricing).
+
+<br />
+
+### New: Altitiny Cloud support
+
+We've partnered up with Altinity to offer a worry-free managed ClickHouse database. Altinity are experts in Clickhouse and data infrastructure and they help large organizations to deploy, manage and scale their PostHog instance.
+
+Find out more about our Altinity integration in our [Marketplace docs](/marketplace/altinity).
+
+<br />
+
+### New: Instance configuration UI
+
+We're significantly improving the way you manage your PostHog instance. Some settings are now configurable directly on the UI, instead of you having to rely on environment variables (e.g. email settings). You can read more about these settings in the [instance settings docs](/docs/self-host/configure/instance-settings) and [environment variables docs](/docs/self-host/configure/environment-variables).
+
+<img src="https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/1_33_0-instance-configuration.png" alt="Screenshot: Instance configuration" />
+
+<br />
+
+
+### Other improvements & fixes
+
+- Improved: Short robust permalinks. When sharing an insight link (e.g. `https://app.posthog.com/insights/BYt1oFdI`), we'll now automatically switch you to the relevant project (if the link is from a different project). You can now be confident, the recipient we'll see the insight you intended.
+- Improved: Smarter events and property filters. We've made a lot of improvements to how we show events and event properties across the app. We'll now only show you the relevant properties for a specific event and we'll signal when an event has not been seen in the last 30 days, so you can create insights faster.
+- Improved: Redesigned share dashboard modal. We've made it simpler and more clear when your dashboard is shared, internally and externally.
+- New: Insight legends. Each insight now offers a legend within each graph to provide clarity to what each line represents. This is particularly useful when sharing screenshots of an insight so the recipient knows what each line represents.
+- Improved: Automatic API documentation. We've implemented [Swagger](https://github.com/swagger-api) to automatically generate API docs to ensure these are always up-to-date.
+- Improved: Performance of Events page. The events page will now load up to 10x faster! We've improved the way queries are performed, and what information is loaded to reduce load times to a minimum. 
+- Plus 290+ improvements & fixes.
+
+### Deprecation & removal notices
+
+No new deprecation notices in this release.
```

Comment:
@yakkomajuri anything we need to add in this array re async migrations?

Reply:
"No new" ones, but is there anything we should remind about? If not, we could just remove the section.

Reply:
Added async migrations information